player/core.h

```diff
@@ -194,7 +194,7 @@ struct ao_chain {
     double start_pts;
     bool start_pts_known;
 
-    bool audio_started;
+    bool delaying_audio_start;
```

Comment:
Name nit but `delayed_audio_start` sounds better imo.

Reply:
That's what I had originally but `delaying` more correctly represents what the bool is for, since it's supposed to flip to false once we're not delaying the audio anymore

Reply:
Fair